mutiny-core/src/lib.rs

```diff
@@ -1126,18 +1127,16 @@ impl<S: MutinyStorage> MutinyWallet<S> {
         amount: Option<u64>,
         labels: Vec<String>,
     ) -> Result<MutinyInvoice, MutinyError> {
-        let federation_ids = self.list_federation_ids().await?;
+        let amt = amount.map_or(Err(MutinyError::InvalidArgumentsError), Ok)?;
```

Comment:
if we're throwing an error if this is None, should jut change the param to not be optional